common/src/test/java/tachyon/heartbeat/HeartbeatContextTest.java

```diff
@@ -36,6 +36,7 @@ public void timerClassesCheckTest() throws Exception {
     checkNotNull(HeartbeatContext.MASTER_TTL_CHECK);
     checkNotNull(HeartbeatContext.WORKER_CLIENT);
     checkNotNull(HeartbeatContext.WORKER_LINEAGE_SYNC);
+    checkNotNull(HeartbeatContext.PIN_LIST_SYNC);
```

Comment:
If you don't mind, can you add `HeartbeatContext.WORKER_BLOCK_SYNC` here a below as well?